README.md

```diff
@@ -2,8 +2,8 @@
 
 [![PyPI](https://img.shields.io/pypi/v/trieste.svg)](https://pypi.org/project/trieste)
 [![License](https://img.shields.io/badge/license-Apache-green.svg)](LICENSE)
-[![Quality checks](https://github.com/secondmind-labs/trieste/actions/workflows/develop-checks.yaml/badge.svg)](https://github.com/secondmind-labs/trieste/actions?query=workflows%3Adevelop-checks)
-[![Docs](https://github.com/secondmind-labs/trieste/actions/workflows/deploy.yaml/badge.svg)](https://github.com/secondmind-labs/trieste/actions/workflows/deploy.yaml)
+[![Deploy](https://img.shields.io/github/actions/workflow/status/secondmind-labs/trieste/deploy.yaml?logo=github&label=Release%20build)](https://github.com/secondmind-labs/trieste/actions/workflows/deploy.yaml)
```

Comment:
what does this refer to - release? perhaps a more descriptive name?

Reply:
Changed (though note that the badge itself was already labelled "Release build").